opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/controllers/ExceptionController.java

```diff
@@ -0,0 +1,15 @@
+package nl.novi.opdrachttechiteasy.controllers;
+
+import nl.novi.opdrachttechiteasy.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+    @ControllerAdvice
+    public class ExceptionController {
+        @ExceptionHandler(RecordNotFoundException.class)
+        public ResponseEntity<String> handleRecordNotFoundException(RecordNotFoundException e) {
+            System.err.println("RecordNotFoundException occurred: " + e.getMessage());
```

Comment:
Heel goed om niet direct de message naar de client terug te sturen, maar de client een "beperkte" message te geven en de echte message te loggen. De complete message kan immers belangrijke informatie blootgeven over de innerlijke werking van jou applicatie, zoals welke klasses of dependencies je gebruikt.